gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -1210,7 +1210,8 @@ function GM:EntityTakeDamage(ent, dmginfo)
 	end
 
 	--Call a second Hook after Damage got processed to output the real Damage that has been dealt
-	return hook.Run("TTT2PostEntityTakeDamage", ent, dmginfo)
+	local tmpdmginfo =  table.Copy(dmginfo)
```

Comment:
This is not possible, noticed it too late....

Reply:
yes, it is probably userinfo, isn't it?